src/app/models/response.ts

```diff
@@ -1,12 +1,30 @@
 /* eslint-disable max-classes-per-file */
-export class Response {
+
+import { HttpTableData } from './http-table-data';
+
+export class Re {
   routesArray: Routes[];
+
+  public id : number;
+
+  public testPlanName: string;
+
+  public loops: number;
+
+  public duration: number;
+
+  public threads: number;
+
+  public rampUp: number;
+
+  public followRedirects: boolean;
+
+  public resultsummaries: HttpTableData [];
```

Comment:
If your class has many fields, it should have a constructor. You can use the `HttpTableData` class as an example to create overloaded constructors if you wish.